starter_files/lib/deck.rb

```diff
@@ -0,0 +1,37 @@
+require_relative "card"
+
+class Deck
+
+  RANKS = [:A, 2, 3, 4, 5, 6, 7, 8, 9, 10, :J, :Q, :K]
+  SUITS = [:clubs, :diamonds, :hearts, :spades]
+
+  def initialize
+
+    @cards = []
+
+    SUITS.each do |suit|
+      RANKS.each {|rank| @cards << Card.new(rank, suit)}
+    end
+
+  end
+
+  def cards_left
+    @cards.length
+  end
+
+  def draw
+    @cards.shift
+  end
+
+  def shuffle
+    prg = Random.new
```

Comment:
`prg` as a variable name is kind of inscrutable. Even though it might be a long name, it's usually preferable to spell out a meaningful name to help the human who's reading the code.